src/test/acceptance/features/steps.js

```diff
@@ -0,0 +1,67 @@
+const { Before, Given, When, Then, After } = require('cucumber')
+const { expect } = require('chai')
+
+const EnterName = require('../../common/page/enter-name')
+const Overview = require('../../common/page/overview')
+const DriverManager = require('../../common/driver/driver-manager')
+const { BASE_URL } = require('../constants')
+
+const driverManager = new DriverManager()
+let enterName
+let overview
+let driver
+
+const LONG_NAME = 'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' + // 100
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' + // 200
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' + // 300
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' + // 400
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' +
+  'This name is way too long' + // 500
+  'This name is way too long'
+
+Before(function () {
+  driver = driverManager.initialise()
+  overview = new Overview(driver)
+  enterName = new EnterName(driver)
+})
+
+After(function () {
+  driver.quit()
+})
+
+Given('I navigate to the HTBHF website and start the application process', async function () {
```

Comment:
We should be starting on the enter-name url as the start page is something that we will not creating or maintaining.

Reply:
Yep, good point, I'll correct that.